misc/python/materialize/cli/mzimage.py

```diff
@@ -84,6 +88,17 @@ def add_image_subcommand(name: str, **kwargs: Any) -> argparse.ArgumentParser:
         subparser.add_argument(
             "image", help="the name of an mzbuild image in this repository"
         )
+        subparser.add_argument(
```

Comment:
`mzcompose` calls these arguments `--mz-build-mode` and `--mz-coverage`, respectively. I'm not sure about the naming scheme of these tools but using the same names here would make it less confusing for me at least. 😅

Reply:
Ah yeah, agree it's confusing! mzcompose does that to distinguish between arguments passed to Docker Compose and arguments passed to mzcompose. It's probably a mistake, and we should strip the "mz-" prefix from those commands. We're slowly moving towards hiding Docker Compose away.